kasa/smart/smartmodule.py

```diff
@@ -138,7 +138,9 @@ def query(self) -> dict:
 
         Default implementation uses the raw query getter w/o parameters.
         """
-        return {self.QUERY_GETTER_NAME: None}
+        if self.QUERY_GETTER_NAME:
+            return {self.QUERY_GETTER_NAME: None}
```

Comment:
If the above change goes into another PR, maybe that PR could also modify the default params to be `{}`?

Reply:
The default params change should definitely be it's own PR :). We'd want to be changing it in all the places at once.